staff/blas-toth/challenges/calculadora/README.md

```diff
@@ -0,0 +1,31 @@
+# Calulator App
+
+## Intro
+
+blah blah blah...
```

Comment:
Missing description :)

Reply:
Thank you, I have added the description.